x/payment/client/cli/wirepayformessage.go

```diff
@@ -52,7 +52,12 @@ func CmdWirePayForMessage() *cobra.Command {
 			}
 
 			// create the MsgPayForMessage
-			pfmMsg, err := types.NewWirePayForMessage(namespace, message, consts.MaxSquareSize)
+			squareSize, err := cmd.Flags().GetUint64("square-size")
```

Comment:
It's also important to note that the user should be able to enter in multiple square sizes, this way, we can generate valid `MsgPayForMessage`s for different potential square sizes. This also makes it more probable block producers will have a viable commitment to the underlying message, and will therefore include the message in the block

Reply:
we should also be using the `FlagSquareSize` constant instead of re-entering the string